### PR TITLE
Feature: button animation, link styles, wip contact us

### PR DESCRIPTION
**Pull request by josadcha**

### CODE REVIEW

File: src/components/Contacts/Contacts.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.contacts {
```
Comment (mod) [inline src/components/Contacts/Contacts.scss:1]: we didn't needed contact form, it's a figma mistake

File: static/img/icons/Phone.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (mod) [inline static/img/icons/Phone.js]: we can't commit js in this folder
you can stick the native .svg content

**Review by mod — CHANGES_REQUESTED** [review]
you can probably rename contact into partners later, you can leave it there, but I'm not sure we will use it

File: src/components/Contacts/Contacts.scss (hunk 1 above)
Comment (josadcha) [reply src/components/Contacts/Contacts.scss:1]: let's keep Contact naming for now, we can delete it later if no need

**Review by mod — APPROVED** [review]

**Pull request merged**